.github/workflows/pre_release_version_bump.yml

```
@@ -0,0 +1,37 @@
+name: Manually Triggered Version Bumps
+
+on:
+  workflow_dispatch:
```

Testing pre_release vb! 
```suggestion
on:
  pull request:
  workflow_dispatch:
```